stack.py

```diff
@@ -0,0 +1,20 @@
+from linked_list import LinkedList
+
+#Stack inherits from LinkedList class
+class Stack(object):
+
+    def __init__(self, iterable=None):
+        if iterable != None:
+            self._linkedList = LinkedList(iterable)
```

Comment:
Isn't your linked list perfectly capable of handling `None` in the place of the `iterable` parameter?  Why duplicate the safety code here?